Tasks/AzureAppServiceManage/task.json

```diff
@@ -164,6 +167,28 @@
             "defaultValue": "",
             "visibleRule": "Action = Install Extensions",
             "helpMarkDown": "Provide the variable name for the local installation path for the selected extension.<br />Note: In case of multiple extensions selected for installation, provide comma separated list of variables that saves the local path for each of the selected extension in the order it appears in the Install Extension field.<br />Example: outputVariable1, outputVariable2"
+        },
+        {
+            "name": "AppInsightsResourceGroupName",
+            "type": "pickList",
+            "label": "App Insights Resource Group Name",
```

Comment:
Check the label with PMs, should the branding be "App Insights" or "Application Insights".